### PR TITLE
Add noPrompt parameter for image verification

### LINKED ISSUE #2491: Signing package prompts user input while it should not
## Version of Singularity:

3.0

### Expected behavior

Public package should return an error, and CLI wrapper should handle it and ask for user input.

### Actual behavior

Public package asks for input itself.

### PR DESCRIPTION
**Pull request by sashayakovtseva**
**Description of the Pull Request (PR):**

Additional parameter in Verify added to disable interactive prompts upon image verification. That is required by CRI.

**This fixes or addresses the following GitHub issues:**

- Closes #2491 

**Before submitting a PR, make sure you have done the following:**

- Read the [Guidelines for Contributing](../CONTRIBUTING.md), and this PR conforms to the stated requirements.
- Added changes to the [CHANGELOG](../CHANGELOG.md) if necessary according to the [Contribution Guidelines](../CONTRIBUTING.md)
- Added tests to validate this PR and tested this PR locally with a `make testall`
- Based this PR against the appropriate branch according to the [Contribution Guidelines](../CONTRIBUTING.md)
- Added myself as a contributor to the [Contributors File](../CONTRIBUTORS.md)

Attn: @singularity-maintainers

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -57,6 +57,7 @@
     - Ralph Castain <rhc@open-mpi.org>
     - Rémy Dernat <remy.dernat@umontpellier.fr>
     - Richard Neuboeck <hawk@tbi.univie.ac.at>
+    - Sasha Yakovtseva <sasha@sylabs.io> <sashayakovtseva@gmail.com>
```
Comment (bauerm97) [inline CONTRIBUTORS.md]: `<sasha@sylabs.io>, <sashayakovtseva@gmail.com>`

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ _The old changelog can be found in the `release-2.6` branch_
 
   - Add http/https protocols for singularity run/pull commands
   - Update to SIF 1.0.2
+  - Add _noPrompt_ parameter to `pkg/signing/Verify` function to enable silent verification
```
Comment (bauerm97) [inline CHANGELOG.md:16]: I would leave this out of the changelog personally. I think we want to reserve changelog for things that change how someone uses Singularity. For example, a change to CLI flags or a behavior change in a command.

**Review by bauerm97 — CHANGES_REQUESTED** [review]
Other than those two very minor things, it looks really good.

File: CHANGELOG.md (hunk 2 above)
Comment (sashayakovtseva) [reply CHANGELOG.md:16]: Even if it is a public package?

Comment (bauerm97) [reply CHANGELOG.md:16]: I see your point. Not too big of a deal IMO so we can just leave it

**Review by bauerm97 — APPROVED** [review]

**Pull request merged**